cms/static/js/spec/video/transcripts/utils_spec.js

```diff
@@ -215,6 +215,39 @@ function($, _, Utils, _str) {
                     });
                 });
             });
+
+            describe('Too long arguments ', function() {
+                var longFileName = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789' +
+                        'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789' +
+                        'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789' +
+                        'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789' +
+                        'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789',
+                    html5LongUrls = (function(videoName) {
+                        var links = [
+                            'http://somelink.com/%s?param=1&param=2#hash',
+                            'http://somelink.com/%s#hash',
+                            'http://somelink.com/%s?param=1&param=2',
+                            'http://somelink.com/%s',
+                            'ftp://somelink.com/%s',
+                            'https://somelink.com/%s',
+                            'https://somelink.com/sub/sub/%s',
+                            'http://cdn.somecdn.net/v/%s',
+                            'somelink.com/%s',
+                            '%s'
+                        ];
+                        return $.map(links, function(link) {
+                            return _str.sprintf(link, videoName);
+                        });
+                    }(longFileName));
+
+                $.each(html5LongUrls, function(index, link) {
+                    it(link, function() {
+                        var result = Utils.parseHTML5Link(link);
+
+                        expect(result.video.length).toBeLessThan(255);
```

Comment:
As we are allowing 100 as max file name size, shouldn't we also check for 100 here?